compiler/tests-compiler/effects_continuations.ml

```diff
@@ -103,79 +103,86 @@ let%expect_test "test-compiler/lib-effects/test1.ml" =
     {|
 
     function exceptions(s, cont){
-     try{var _t_ = runtime.caml_int_of_string(s), n = _t_;}
-     catch(_x_){
-      var _p_ = caml_wrap_exception(_x_);
-      if(_p_[1] !== Stdlib[7]){
+     try{var _w_ = runtime.caml_int_of_string(s), n = _w_;}
+     catch(_A_){
+      var _s_ = caml_wrap_exception(_A_);
+      if(_s_[1] !== Stdlib[7]){
        var raise$1 = caml_pop_trap();
-       return raise$1(caml_maybe_attach_backtrace(_p_, 0));
+       return raise$1(caml_maybe_attach_backtrace(_s_, 0));
       }
       var n = 0;
      }
      try{
       if(caml_string_equal(s, cst$0))
        throw caml_maybe_attach_backtrace(Stdlib[8], 1);
-      var _s_ = 7, m = _s_;
+      var _v_ = 7, m = _v_;
      }
-     catch(_w_){
-      var _q_ = caml_wrap_exception(_w_);
-      if(_q_ !== Stdlib[8]){
+     catch(_z_){
+      var _t_ = caml_wrap_exception(_z_);
+      if(_t_ !== Stdlib[8]){
        var raise$0 = caml_pop_trap();
-       return raise$0(caml_maybe_attach_backtrace(_q_, 0));
+       return raise$0(caml_maybe_attach_backtrace(_t_, 0));
       }
       var m = 0;
      }
      runtime.caml_push_trap
-      (function(_v_){
-        if(_v_ === Stdlib[8]) return cont(0);
+      (function(_y_){
+        if(_y_ === Stdlib[8]) return cont(0);
         var raise = caml_pop_trap();
-        return raise(caml_maybe_attach_backtrace(_v_, 0));
+        return raise(caml_maybe_attach_backtrace(_y_, 0));
        });
      if(! caml_string_equal(s, cst))
       return caml_cps_call2
               (Stdlib[79],
                cst_toto,
-               function(_u_){caml_pop_trap(); return cont([0, [0, _u_, n, m]]);});
-     var _r_ = Stdlib[8], raise = caml_pop_trap();
-     return raise(caml_maybe_attach_backtrace(_r_, 1));
+               function(_x_){caml_pop_trap(); return cont([0, [0, _x_, n, m]]);});
+     var _u_ = Stdlib[8], raise = caml_pop_trap();
+     return raise(caml_maybe_attach_backtrace(_u_, 1));
     }
     //end
     function cond1(b, cont){
-     function _o_(ic){return cont([0, ic, 7]);}
+     function _p_(ic){return cont([0, ic, 7]);}
      return b
-             ? caml_cps_call2(Stdlib[79], cst_toto$0, _o_)
-             : caml_cps_call2(Stdlib[79], cst_titi, _o_);
+             ? caml_cps_call2
+               (Stdlib[79], cst_toto$0, function(_q_){return _p_(_q_);})
```

Comment:
Why do we have this diff ?

Reply:
Maybe some inlining optim needs to be updated ?

Reply:
For the CPS transformation, we are splitting blocks if a function call is followed by some instructions.
Since I added an event instruction, this resulted in more splitting. But it seems fine to just skip events in this case.